apps/vercel/frontend/src/clients/Vercel.ts

```diff
@@ -28,57 +29,56 @@ export default class VercelClient implements VercelAPIClient {
     });
   }
 
-  async checkToken(): Promise<CheckTokenResponse> {
-    const res = await fetch(`${this.baseEndpoint}/v5/user/tokens`, {
-      headers: this.buildHeaders(),
-      method: 'GET',
-    });
+  private buildTeamIdQueryParam(teamId?: string): string {
+    return teamId ? `&teamId=${teamId}` : '';
+  }
 
-    return { ok: res.ok };
+  async checkToken(): Promise<GetToken> {
+    const res = await this.getToken();
+    const { token } = await res.json();
+
+    if (!res.ok) throw new Error(errorTypes.INVALID_TOKEN);
```

Comment:
Should we move this one line above? In case eg there wasn't a json response? Not sure it matters.

Reply:
Yeah good catch 🦅 , have adjusted this!